android/app/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,7 @@
     android:versionName="1.0">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" /> 
```

Comment:
I assume this is a natural requirement for applications, but why is it necessary now?

Reply:
I installed the rn-fetch-blob package, maybe it added it.